### ISSUE TITLE
HermitCrab Step File

### ISSUE THREAD

**Issue by DroneMang** [bigtreetech/HermitCrab#2]
Hi,

I see you have the stl for the regular hermit crab.  Would it be possible to post a step file?  I realize you may not want to post it in full precision to prevent copying, if you could post a simplified model or less accurate one, even without the pins and latch, I could use it in the CAD model of my printer.  I just need the mounting points and overall dimensions.  

Can you also post a .step file for the v-slot roller bracket and belt attachment bracket for the linear rails? 

Thanks!

**Comment by Stefkeys** [comment]
Agreed! It would be nice to be able to make some accurate measurements for cooling ducts like the Afterburner

**Comment by chris-it-is** [comment]
The BTT support team just replied to my question about step files for the hermit crab.
There are actually already some step files available for download on their homepage:
https://bigtree-tech.com/pages/download

You can find a download section labeled "Hermit Crab Series" with two step files there.
